src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow
```

Comment:
Same as for `ReactServerRenderingTransaction.js`, flow passes her so it's apparently sufficiently typed. Adding flow has two effects I'd like to keep if you don't mind:

1. Performing typechecking in the file itself (as well as checking nullability for example)
2. Forcing to provide flow annotations on exposed function signatures (which is the case there! => `ReactReconcileTransaction(useCreateElement: boolean)`)

I would "vote" to keep it, but I understand if you want it to disappear and wait for a PR that will be correctly typechecking all of that, but better!